sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -3870,6 +3870,17 @@ class DataSourceV2SQLSuiteV1Filter
     }
   }
 
+  test("EXPLAIN") {
```

Comment:
Is there a reason why this is added to `DataSourceV2SQLSuiteV1Filter` test suite?

Shall we move this to `DataSourceV2SQLSuite` to test in both `DataSourceV2SQLSuiteV1Filter` and `DataSourceV2SQLSuiteV2Filter` simply because the main change is on `DataSourceV2RelationBase`?

Reply:
No reason, I simply noticed we didn't have any test specific to `DataSourceV2SQLSuite`.
It makes sense to move, done.